packages/skott/src/skott.ts

```diff
@@ -58,7 +59,8 @@ const defaultConfig = {
   circularMaxDepth: Number.POSITIVE_INFINITY,
   dependencyTracking: {
     thirdParty: false,
-    builtin: false
+    builtin: false,
+    typeOnly: true
```

Comment:
As you correctly mentioned, `types` import are not really problematic in the context of dependency cycles, don't you think it should be the default behavior to ignore them instead?

Reply:
Well the primary use case is not really visualization (even if it seems to be when using the cli), but it's more about building a graph structure based on runtime dependencies. But you're right it might be interesting to see all edges related to TypeScript type-level stuff in the first place - otherwise it would not be really helpful to use skott directly on TypeScript files (as it would have most likely produced the same result running analysis from transpiled JavaScript files).
Let's stay on the default you introduced 🙂